.devcontainer/devcontainer.json

```diff
@@ -47,7 +47,7 @@
 				"remote.otherPortsAttributes": {
 					"onAutoForward": "ignore"
 				},
-				"dotnet.defaultSolution": "Aspire.sln"
```

Comment:
I believe devcontainer will respect the settings in the local .vscode directory so we can just have all the customizations there so they work for non-DevContainer users as well.